### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by mitchhentges**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [upcoming]
+## [0.15.0] - 2019-02-11
```
Comment (escapewindow) [inline CHANGELOG.md]: We may have agreed on 1.0.0, but it doesn't have to be this PR.

Comment (mitchhentges) [reply CHANGELOG.md]: I want to wait for [this RFC](https://github.com/mozilla-releng/releng-rfcs/pull/14) to land before we bump to major versions.

**Comment by mitchhentges** [comment]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (escapewindow) [reply CHANGELOG.md]: This means that a conversation that [was resolved](https://bugzilla.mozilla.org/show_bug.cgi?id=1524248#c16) is hung up on a larger process change. I don't think the RFC hard blocks.

Comment (mitchhentges) [reply CHANGELOG.md]: :+1:, I'll bump to `1.0.0`

**Pull request merged**